detox/test/e2e/28.visibility.test.js

```diff
@@ -24,6 +24,16 @@ describe('Visibility Cases', () => {
     });
   });
 
+  it(':ios: should be able to dismiss the keyboard by scrolling down in the ScrollView', async () => {
```

Comment:
This won't magically work on its own. It requires that the screen add support for this in its code.

Reply:
Got it. I've removed the test until the better times come. 😄